thyrel-web/src/components/styles/Box.tsx

```diff
@@ -0,0 +1,134 @@
+import styled from '@emotion/styled';
+
+// TODO : add multi
+// const multi: number = 8;
+
+type BoxType = {
+  display?: 'flex';
+  flexDirection?: 'row' | 'row-reverse' | 'column' | 'column-reverse';
+  flexWrap?: 'nowrap' | 'wrap' | 'wrap-reverse';
+  width?: number | string;
+  height?: number | string;
+  justifyContent?:
+    | 'flex-start'
+    | 'flex-end'
+    | 'center'
+    | 'space-between'
+    | 'space-around'
+    | 'space-evenly'
+    | 'start'
+    | 'end'
+    | 'left'
+    | 'right';
+  alignItems?:
+    | 'stretch'
+    | 'flex-start'
+    | 'flex-end'
+    | 'center'
+    | 'baseline'
+    | 'first baseline'
+    | 'last baseline'
+    | 'start'
+    | 'end'
+    | 'self-start'
+    | 'self-end'
+    | 'safe'
+    | 'unsafe';
+  alignContent?:
+    | 'flex-start'
+    | 'flex-end'
+    | 'center'
+    | 'space-between'
+    | 'space-around'
+    | 'space-evenly'
+    | 'stretch'
+    | 'start'
+    | 'end'
+    | 'baseline'
+    | 'first baseline'
+    | 'last baseline'
+    | 'safe'
+    | 'unsafe';
+  margin?: number | string;
+  m?: number | string;
+  mb?: number | string;
+  ml?: number | string;
+  mt?: number | string;
+  mr?: number | string;
+  padding?: number | string;
+  p?: number | string;
+  pb?: number | string;
+  pl?: number | string;
+  pt?: number | string;
+  pr?: number | string;
+};
+
+const Box = styled.div<BoxType>(
```

Comment:
Ajout d'un composant qui gère tout ce qui est du `Flex` qu'on vas **beaucoup** utilisé dans notre app :
https://css-tricks.com/snippets/css/a-guide-to-flexbox/